### ISSUE TITLE
A11y_SQLAzureDataStudio_ImportBacpac_Summary Screen_ScreenReader: Alert is announced when "Summary" step is opened.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#9171]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11y_SQLAzureDataStudioBenchmark;#A11yMAS;#A11ySev2;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#DesktopApp;#Win32;#MAS1.3.1;#JAWS;#NVDA;  

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.16.0-insider
OS: Windows_NT x64 10.0.18363

**Additional Details:**
Screen reader: NVDA, JAWS
MAS References:MAS1.3.1

**Repro Steps:**
1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Extensions from left menu.
4. Search & install Dacpac extension.
5. Navigate to Connections from left menu.
6. Expand the server connected in step 2.
7. Right click an on prem database and select Data-tier application wizard from Context menu.
8. Select third radio button.
9. Launch screen reader as NVDA
10. Navigate till next button & hit enter.
11. Provide all the details & hit enter.

**Actual:**
When the page "Summary" is opened the screen reader announces alert before the word summary.

**Same issue found with JAWS**.

**Same issue in the following page:**
1. Select an Operation
2. Export Bacpac-Summary screen
3. Import Bacpac Settings  
4. Export Bacpac Settings
5.SQL Server Big Data Cluster-Summary

**Expected:**
The announcement of the word "alert" should be not made.

**User Impact:**
Screen reader users will think there is a mandatory field or requirement at summary.

**Attachment link for Reference :**
[A11y_SQLAzureDataStudio_ImportBacpac_Summary Screen_ScreenReader9171.zip](https://github.com/microsoft/azuredatastudio/files/4322666/A11y_SQLAzureDataStudio_ImportBacpac_Summary.Screen_ScreenReader9171.zip)

**Comment by alanrenmsft** [comment]
we probably shouldn't use alert role there.

**Comment by alanrenmsft** [comment]
let's try use aria-live="polite"

**Comment by kisantia** [comment]
aria-live="polite" didn't announce the title. Need to find another way to get the page title announced

**Comment by udeeshagautam** [comment]
Kim to follow up with Charles for this one. if there is not other solution we can take this to the SME review.

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
